libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Utilities/CSFDefines.m

```diff
@@ -26,3 +26,4 @@
 
 NSString * const CSFValidationErrorDomain = @"CSFValidationErrorDomain";
 NSString * const CSFNetworkInitializedNotification = @"CSFNetworkInitializedNotification";
+NSString * const CSFNetworkLogIdentifier = @"com.salesforce.MobileSDK.Network";
```

Comment:
Log identifier starts with `com.salesforce.MobileSDK` so that other MobileSDK libraries can add their own logging identifiers under a shared prefix.  This way, a developer can set the log level for the entire mobile SDK at once using:

```
[[SFLogger sharedInstance] setLogLevel:SFLogLevelDebug forIdentifiersWithPrefix:@"com.salesforce.MobileSDK"]
```